docs-next/pages/apis/access-control.mdx

```diff
@@ -12,9 +12,15 @@ export default config({
   lists: createSchema({
     ListKey: list({
       fields: {
-        fieldName: text({ access: { /* ... */ }, }),
```

Comment:
I'm not sure why these got changed, looks like our prettier ignore might not be configured correctly? or this is maybe something that happened locally

Because this shows up w/ formatting in the docs though, we'll want to roll all the mdx changes back in this PR

@gwyneplaine do you know if we've made any changes to the repo config that would have caused this to happen?

Reply:
@JedWatson not aware of any prettier config changes, however these single-line formatted comments were manual commits via suggestion iirc.

I honestly i think we should stop fighting prettier at this point cc @timleslie

Reply:
@gwyneplaine agree on not fighting prettier generally

In this case I'd rather we tell it to ignore `mdx` files entirely if we can't just tell it to ignore `tsx` in `mdx`

This formatting ends up in the docs, and all the extra line breaks hurt readability much more than the benefit having prettier runs on our examples adds, at least at this point imo

Reply:
@JedWatson how do you want me to resolve? Just checkout the MDX files from origin/master and push back up?

Reply:
Reverted all to `origin/master`